examples/fluids/src/grid_anisotropy_tensor.c

```diff
@@ -29,18 +28,13 @@ PetscErrorCode GridAnisotropyTensorProjectionSetupApply(Ceed ceed, User user, Ce
   // -- Create DM for Anisotropic tensor L^2 projection
   grid_aniso_proj->num_comp = 7;
   PetscCall(DMClone(user->dm, &grid_aniso_proj->dm));
-  PetscCall(DMGetDimension(grid_aniso_proj->dm, &dim));
   PetscCall(PetscObjectSetName((PetscObject)grid_aniso_proj->dm, "Grid Anisotropy Tensor Projection"));
 
   {  // -- Setup DM
-    PetscFE      fe;
     PetscSection section;
-    PetscInt     q_order = user->app_ctx->degree + user->app_ctx->q_extra;
-    PetscCall(PetscFECreateLagrange(PETSC_COMM_SELF, dim, grid_aniso_proj->num_comp, PETSC_FALSE, user->app_ctx->degree, q_order, &fe));
-    PetscCall(PetscObjectSetName((PetscObject)fe, "Grid Anisotropy Tensor Projection"));
-    PetscCall(DMAddField(grid_aniso_proj->dm, NULL, (PetscObject)fe));
-    PetscCall(DMCreateDS(grid_aniso_proj->dm));
-    PetscCall(DMPlexSetClosurePermutationTensor(grid_aniso_proj->dm, PETSC_DETERMINE, NULL));
+    PetscCall(DMSetupByOrder_FEM(PETSC_TRUE, PETSC_TRUE, user->app_ctx->degree, 1, user->app_ctx->q_extra, 1, &grid_aniso_proj->num_comp,
+                                 grid_aniso_proj->dm));
+    PetscCall(DMSetupByOrder_Finalize(PETSC_TRUE, grid_aniso_proj->dm));
```

Comment:
Maybe a helper for when these two are back-to-back?

Reply:
Yeah, I'm fine with that. Suggestions on name?

Reply:
Maybe `DMSetupByOrder_FEM_NoBC`?

Reply:
Depends upon if we intend this to stick around long term. 

Perhaps
`DMSetupByOrderBegin_FEM`
`DMSetupByOrderEnd_FEM`
`DMSetupByOrder_FEM` (the two together)

or something along those lines

Reply:
I don't *intend* this to stick around long term. But intent is different than reality sometimes.

I'm fine going that route and then changing it in the future. I don't think the migration between the two will be too difficult.

Reply:
Done.